crates/ark/src/lsp/completions/sources/unique/file_path.rs

```diff
@@ -33,11 +35,7 @@ pub(super) fn completions_from_string_file_path(
     // NOTE: This includes the quotation characters on the string, and so
     // also includes any internal escapes! We need to decode the R string
     // before searching the path entries.
-    let token = context
-        .document
-        .contents
-        .node_slice(&context.node)?
-        .to_string();
+    let token = context.document.contents.node_slice(&node)?.to_string();
```

Comment:
Can't use node directly from `context` anymore, it is likely a child of the `String` node we care about